### DISCUSSION TITLE
Input hint is overlayed if there is the background color

### DISCUSSION THREAD

**Discussion by 549393092** [quasarframework/quasar#16030] [General - Components / Directives / etc] [answered]
Dear Quasar team,

I did a demo to explain this issue, could you please have a look?
https://codepen.io/Stanley-549393092/pen/oNQWrXQ?editors=1010
As you can see, if without the background style, it can display the hint totally.
However, if with the background style, it's overlayed.
How to display the hint totally even if there is the background color?

**Comment by pdanpdan** [comment, marked as answer]
This is normal - the cells are ordered from left to right and latter cells are displayed over what overflows from former ones
Add a `style="z-index: 1"` to your cell
https://codepen.io/pdanpdan/pen/abQwoZp?editors=1010

**Reply by 549393092** [reply]
@pdanpdan Yes, it works. Thanks a lot!
